packages/client/test/test-utils/utils.ts

```diff
@@ -233,7 +234,8 @@ export class LeaksDetector {
 
     private counter = CounterId(this.id, { maxPrefixes: 1024 })
 
-    add(name: string, obj: any) {
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+    add(name: string, obj: any): void {
```

Comment:
Class should probably implement a generic type to capture values' type.

Reply:
It is just an test utility